platform/claims.go

```diff
@@ -22,6 +23,10 @@ func (o Profile) GetName() string {
 	return ProfileName
 }
 
+func (o Profile) GetNameLegacy() string {
```

Comment:
A profile (`psatoken.IProfile`) cannot have a legacy name. What we have here is a legacy profile. Instead of this, should create a separate `LegacyProfile` and register it alongside `Profile` inside `init()` below.

Reply:
Thanks for spotting this.  It's a leftover from an early attempt.

I have removed it in [f8d052d](https://github.com/veraison/ccatoken/pull/32/commits/f8d052d7606ce7bf3f6dc42b71aaff07c5b409ca)

Reply:
Cool, but the legacy profile should still be created and registred (this will allow generic unmarshaler to recognise legacy tokens).

Reply:
Not sure I am taking an unwarranted shortcut (likely so), but from the generic unmarshaler perspective there is no difference between legacy and new because there are no new codepoints and the CBOR encoding is exactly the same.  In fact, the two profile identifiers are essentially synonyms.

Reply:
Sure, but they are still different profiles (have different profile name strings in the claims). In order for the legacy profile string to be recognised by the generic unmarshallers, it needs to be registered. The `GetClaims()` for the legacy profile can be the same as the the `Profile`.